src/NativeSerializer.js

```diff
@@ -35,4 +38,12 @@ const serializeForNativeLayer = (map, maxDepth = 10, depth = 0, seen = new Set()
   return output
 }
 
+const extractErrorDetails = (err) => {
+  const {message, stack} = err
+  return {
+    message: message,
```

Comment:
there's nice new-ish js shortand for this:

```js
return { message, stack }
```

but you should also return the `name` and potentially the `code` property (this is only relevant to DOMexceptions, can you get them in RN? 🤔):

Reply:
From what I've been reading the `message` and `stack` are the only guarantees.  I don't want to be dumping a whole load of `undefined`s into the metadata, so left out everything else.

I could add test whether properties are assigned and extract them if so?